app/main.py

```diff
@@ -1 +1,25 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    comparison_list = command.split()
+    if len(comparison_list) == 3:
+        command_set = command.split()
+        if command_set[0] != "mv":
+            raise Exception(
+                f"You are allowed to use only 'mv' command! "
+                f"Meanwhile you have inputted {command_set[0]}!"
```

Comment:
what is command_set[0]? create a separated variable for this command and call it on demand
```
    comparison_list = command.split()
    if len(comparison_list) == 3:
        command_set = command.split()
```
don't do splitting twice, use 1 variable